src/arrayMethodSort.js

```diff
@@ -4,8 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (element1, element2) =>
+      String(element1) > String(element2),
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 0; j < this.length - i - 1; j++) {
+        if (compareFunction(this[j], this[j + 1]) > 0) {
```

Comment:
The comparison function should return a number. If it's expected to return something other than a number, the result should be coerced to a number to ensure correct behavior. For example, you could use `Number(compareFunction(this[j], this[j + 1])) > 0` to coerce the result to a number.